launchable/commands/record/build.py

```diff
@@ -119,3 +126,12 @@ def build(ctx, build_name, source, max_days, no_submodules,
             raise e
         else:
             print(e)
+
+    org, workspace = get_org_workspace()
+    click.echo(
+        "Launchable recorded build {} to workspace {}/{} with commits {} repositories\n".format(build_name, org, workspace, len(uniq_submodules)))
```

Comment:
Technically, `{}/{}` is not a workspace but we call it a "tenant" internally but I'm not sure we should surface it here or not

Reply:
I'm not sure too... However, I think the user can easy for understand `workspace` than `tenant`.
@awilkes What do you think about it?

Reply:
I think customers will understand `workspace {}/{}` here, similarly to the way they would understand a phrase like "the `launchableinc/cli` repository"

Reply:
Two more things:

1. we're missing `from`, i.e. `with commits from {} repositories`
2. if repository count = 1, then it should be `with commits from {} repository`

Reply:
What happens if we recorded 0 commits?

Reply:
I fixed some messages https://github.com/launchableinc/cli/pull/283/commits/ad1edf0b64bf8193224a4f399cd2fd8690d9cdfc

And, at least one commit will be sent, so we don't need to care 0 commit
e.g)
```
$  launchable record build --name cli
Processed 2 commits
Launchable recorded build cli to workspace launchableinc/cli with commits from 1 repository

| Name   | Path   | HEAD Commit                              |
|--------|--------|------------------------------------------|
| .      | .      | ad1edf0b64bf8193224a4f399cd2fd8690d9cdfc |

$  launchable record build --name cli
Processed 1 commits
Launchable recorded build cli to workspace launchableinc/cli with commits from 1 repository

| Name   | Path   | HEAD Commit                              |
|--------|--------|------------------------------------------|
| .      | .      | ad1edf0b64bf8193224a4f399cd2fd8690d9cdfc |

$ launchable record build --name cli
Processed 1 commits
Launchable recorded build cli to workspace launchableinc/cli with commits from 1 repository

| Name   | Path   | HEAD Commit                              |
|--------|--------|------------------------------------------|
| .      | .      | ad1edf0b64bf8193224a4f399cd2fd8690d9cdfc |
```